contracts/Controller.sol

```diff
@@ -93,6 +93,23 @@ contract Controller is ReentrancyGuard, Ownable {
         uint256 vaultId,
         uint256 amount
     );
+    /// @notice emits an event when a short otoken get burned from a vaukt
+    event ShortOtokenBurned(
+        address indexed otoken,
+        address indexed AccountOwner,
+        address indexed from,
+        uint256 vaultId,
+        uint256 amount
+    );
+    /// @notice emits an event when a exercise action execute
+    event Exercise(
+        address indexed otoken,
+        address indexed from,
+        address indexed exerciser,
```

Comment:
Currently these 2 means msg.sender and exerciser and it could be quite confusing. Maybe change the name exerciser to owner?

Reply:
Updated the names in the event+parse args.